keps/973-workload-priority/README.md

```diff
@@ -115,12 +116,21 @@ type WorkloadPriorityClass struct {
 	Value int32   `json:"value,omitempty"`
 	Description string `json:"description,omitempty"`
 }
+```
+
+Also `priorityClassSource` field is added to `WorkloadSpec`.
```

Comment:
Explain what this is and possible values, or add a link to the sections that explain it.